clients/spark-extensions-base/src/main/scala/org/apache/spark/sql/execution/datasources/v2/NessieUtils.scala

```diff
@@ -61,11 +61,14 @@ object NessieUtils {
       .map(x => x.replaceAll("`", ""))
       .map(x => {
         try {
-          LocalDateTime.parse(x).atZone(ZoneOffset.UTC).toInstant
+          ZonedDateTime.parse(x).toInstant
```

Comment:
Apologies for the late response. 

Spark-3.3 introduced native syntax for time travel (similar to what delta has) and even they don't support timezones in the SQL time travel syntax.

So, just to be unified with spark. Can we use 
`SET TIME ZONE ` 'America/Los_Angeles';
or 
`spark.conf.set("spark.sql.session.timeZone", "UTC")`
syntax for setting the timezones?

for both of the above cases we can use the `spark.conf().get("spark.sql.session.timeZone")`
to get the currently set timezone and convert the timestamp here?

@snazy, @tomekl007 : WDYT?

Reply:
We discussed that during our call and decided to not do it. We want to use a similar approach to Iceberg and Delta late and rely on explicit time zone passing.

Reply:
ok.